custom/icds_reports/reports/utils.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import six
+
+
+def get_color_with_green_positive(val):
```

Comment:
A docstring explaining how strings are handled would be nice for both of the functions in this file.

```suggestion
def get_color_with_green_positive(val):
    """Get color for value; postive -> green, other -> red

    :param val: Numeric value or text. All text values are interpreted
    as non-positive, and will return `'red'`.
    :returns: Text: `'red'` or `'green'`.
    """
```


Also, is `val` expected to be a particular value (empty string) when it is `text_type`? The title implies that it's an empty string, but this code interprets any `text_type` value (even `'42'`) as negative (red). Should there be an `assert val == '', val` in the `if`-block?